zeppelin-server/src/test/java/org/apache/zeppelin/rest/NotebookRestApiTest.java

```diff
@@ -1187,4 +1189,72 @@ void testRunWithServerRestart() throws Exception {
       }
     }
   }
+
+  @Test
+  void testGetJobList_whenJobManagerDisabled() throws IOException {
+    boolean originalFlag = disableJobManagerAndBackupFlag();
+    String expectedErrorMessage = "Job Manager is disabled in the current configuration.";
+    CloseableHttpResponse response = null;
+
+    try {
+      response = httpGet("/notebook/jobmanager/");
+      assertEquals(Response.Status.FORBIDDEN.getStatusCode(),
+          response.getStatusLine().getStatusCode(),
+          "Response status should be 403 Forbidden");
+
+      String jsonResponse = EntityUtils.toString(response.getEntity(), StandardCharsets.UTF_8);
+      Map<String, Object> parsedResponse = gson.fromJson(jsonResponse,
+          new TypeToken<Map<String, Object>>() {}.getType());
+
+      assertEquals("FORBIDDEN", parsedResponse.get("status"));
+      assertEquals(expectedErrorMessage, parsedResponse.get("message"));
+
+    } finally {
+      if (response != null) {
+        response.close();
+      }
+      restoreJobManagerFlag(originalFlag);
+    }
+  }
+
+  @Test
+  void testGetUpdatedJobList_whenJobManagerDisabled() throws IOException {
+    boolean originalFlag = disableJobManagerAndBackupFlag();
+    String expectedErrorMessage = "Job Manager is disabled in the current configuration.";
+    CloseableHttpResponse response = null;
+
+    try {
```

Comment:
Please use the try-with-resources pattern, which is more elegant.
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html